### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by successfulbarrier** [IDEA-Research/GroundingDINO#399]
Where can I find the training code? Is there code for the training part of the project?

**Comment by shreyass-ranganatha** [comment]
mmdetection allows training [here](https://github.com/open-mmlab/mmdetection/blob/main/configs/grounding_dino/README.md), but personally, mmdetection is a pain to work with, I really want the training code for this repo as well

Huggingface's transformers' GroundingDINO has a loss function implementation but I'm unsure if there are any caveats to training it on there
